frontend/src/EditAnnotationCampaign.tsx

```diff
@@ -140,7 +139,7 @@ class EditAnnotationCampaign extends Component<EACProps, EACState> {
       undefined : { annotation_goal: this.state.new_ac_annotation_goal};
     const res = Object.assign({}, params, annotation_goal);
 
-    const postUrl = POST_ANNOTATION_CAMPAIGN_API_URL.replace('ID', this.state.campaign_id.toString());
+    const postUrl = POST_ANNOTATION_CAMPAIGN_API_URL.replace('ID', this.state.campaign_id.toFixed());
```

Comment:
To fully answer your comment:

- I prefer to use `toFixed()` over `toString()` to be sure that the formatted string is exactly what I want. But I admit that it certainly is overkill, `toString()` is supposed to do the job right for integers.
- I removed useless `toString()` uses where the route matching was used (`match.params.xxx_id`) because route params are always strings. Here, I could have used it too. I chosed to keep the previous use of the id retrieved by the first API call, but again it does not change much.